csharp/Managed/Events/FloatEventArgs.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using LibPDBinding.Managed.Data;
+
+namespace LibPDBinding.Managed.Events
+{
+	public class FloatEventArgs : EventArgs
+	{
+		public string Receiver { get; private set; }
+
+		public Float Float { get; private set; }
```

Comment:
This could also contain a C# `float` instead of the `IAtom` encapsulation, see comment to ListEventArgs.cs.